src/ActorMultiVertex.cpp

```diff
@@ -289,9 +291,34 @@ void ActorMultiVertex::DrawPrimitives()
 	}
 }
 
-void ActorMultiVertex::DrawInternal( const AMV_TweenState *TS )
-{
+using DrawFunction = std::function<void(const RageSpriteVertex*, int, float)>;
```

Comment:
I do think this is less legible and that the switch is actually preferable. For relatively small and dense cases (like in the case of enums), jump tables are often used which are pretty efficient.

These global maps also aren't trivially destructible so this breaks the style guide.

Here and everywhere below.